CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.9.0] - 2015-12-21
+### Added
+- Add package.json for npm support
```

Comment:
@jeremiak did we completely figure out npm support? Like could I currently do npm install and `@import 'wds'` and it'll work?